apps/meteor/tests/e2e/page-objects/fragments/home-content.ts

```diff
@@ -1,9 +1,16 @@
 import fs from 'fs/promises';
+import { resolve, join, relative } from 'node:path';
 
 import type { Locator, Page } from '@playwright/test';
 
 import { expect } from '../../utils/test';
 
+const FIXTURES_PATH = relative(process.cwd(), resolve(__dirname, '../../fixtures/files'));
+
+export function getFilePath(fileName: string): string {
+	return join(FIXTURES_PATH, fileName);
+}
+
```

Comment:
I know this is unrelated but I'm tired of running the tests from the root of the repository and not having it work without manually adjusting the paths.